docs/tutorials/getting-started.md

```diff
@@ -106,42 +99,22 @@ However, in your own project, you may wish to pin [Haskell.nix][] (as
 you would pin Nixpkgs). This will make your builds reproducible, more
 predictable, and faster (because the fixed version is cached).
 
-One way of doing this is to use `nix-prefetch-git` to get a JSON file
-with a revision and SHA-256 hash of [Haskell.nix][].
-
-```
-$ nix-prefetch-git --quiet https://github.com/input-output-hk/haskell.nix | tee haskell-nix-src.json
-{
-  "url": "https://github.com/input-output-hk/haskell.nix",
-  "rev": "f1a94a4c82a2ab999a67c3b84269da78d89f0075",
-  "date": "2019-06-05T01:06:12+00:00",
-  "sha256": "0ggxsppjlb6q6a83y12cwgrdnqnw1s128rpibgzs5p1966bdfqla",
-  "fetchSubmodules": false
-}
-```
-
-(The `tee` command is just to show you the result.)
-Use the following expression to import that version:
+Straightforward way of doing this is to change branch name to a revision.
 
 ```nix
-{ nixpkgs ? <nixpkgs> }:
+{ defaultCompilerNixName ? "ghc865"
 
-let
-  spec = builtins.fromJSON (builtins.readFile ./haskell-nix-src.json);
-  haskell-nix-src = (import nixpkgs {}).fetchgit {
-    name = "haskell-lib";
-    inherit (spec) url rev sha256 fetchSubmodules;
-  };
-in
-  import nixpkgs (import haskell-nix-src)
+# Fetch the latest haskell.nix and import its default.nix
+, haskellNix ? import (builtins.fetchTarball "https://github.com/input-output-hk/haskell.nix/archive/f1a94a4c82a2ab999a67c3b84269da78d89f0075.tar.gz") {}
```

Comment:
`defaultCompilerNixName` should really be passed on here with `{ inherit defaultCompilerNixName; }` or perhaps we could leave it out?

Reply:
How does Cabal project deal without default GHC?

Reply:
It uses the default default (ghc865)

Reply:
Alright I removed it.